imap_processing/idex/l1/idex_l1.py

```diff
@@ -886,75 +540,76 @@ def process(self) -> xr.Dataset:
                 name=var,
                 data=[value],
                 dims=("epoch"),
-                attrs=dataclasses.replace(
-                    idex_cdf_attrs.trigger_base,
-                    catdesc=trigger_description.notes,
-                    fieldname=trigger_description.field,
-                    var_notes=trigger_description.notes,
-                    validmax=2**trigger_description.num_bits - 1,
-                    label_axis=trigger_description.label,
-                    units=trigger_description.units,
-                ).output(),
+                attrs=idex_attrs.get_variable_attributes(trigger_description.name),
             )
 
         # Process the 6 primary data variables
         tof_high_xr = xr.DataArray(
             name="TOF_High",
             data=[self._parse_high_sample_waveform(self.TOF_High_bits)],
-            dims=("epoch", "Time_High_SR_dim"),
-            attrs=idex_cdf_attrs.tof_high_attrs.output(),
+            dims=("epoch", "time_high_ssr_dim"),
```

Comment:
I think there is a type here
```suggestion
            dims=("epoch", "time_high_sr_dim"),
```